connector/src/test/scala/com/microsoft/kusto/spark/KustoTestUtils.scala

```diff
@@ -138,18 +139,46 @@ private [kusto] object KustoTestUtils {
                           spark: SparkSession) : Boolean = {
 
     val conf = Map[String, String](
-      KustoSourceOptions.KUSTO_AAD_APP_ID -> kustoConnectionOptions.AppId,
-      KustoSourceOptions.KUSTO_AAD_APP_SECRET -> kustoConnectionOptions.AppKey,
-      KustoSourceOptions.KUSTO_AAD_AUTHORITY_ID -> kustoConnectionOptions.Authority
+      KustoSourceOptions.KUSTO_AAD_APP_ID -> kustoConnectionOptions.appId,
+      KustoSourceOptions.KUSTO_AAD_APP_SECRET -> kustoConnectionOptions.appKey,
+      KustoSourceOptions.KUSTO_AAD_AUTHORITY_ID -> kustoConnectionOptions.authority
     )
 
     val tableRows = spark.read
-      .kusto(s"https://${kustoConnectionOptions.Cluster}.kusto.windows.net",
-      kustoConnectionOptions.Database, tableName, conf)
+      .kusto(s"https://${kustoConnectionOptions.cluster}.kusto.windows.net",
+      kustoConnectionOptions.database, tableName, conf)
 
     tableRows.count() == tableRows.intersectAll(expectedRows).count()
 
   }
 
-  case class KustoConnectionOptions(Cluster: String, Database: String, AppId: String, AppKey: String, Authority: String)
+  private def getSystemVariable(key:String) = {
+    var value = System.getenv(key)
+    if (value == null) {
+      value = System.getProperty(key)
+    }
+    value
+  }
+
+  def getSystemTestOptions: KustoConnectionOptions = {
+    var appId: String = KustoTestUtils.getSystemVariable(KustoSinkOptions.KUSTO_AAD_APP_ID)
+    var appKey: String = KustoTestUtils.getSystemVariable(KustoSinkOptions.KUSTO_AAD_APP_SECRET)
+    var authority: String = KustoTestUtils.getSystemVariable(KustoSinkOptions.KUSTO_AAD_AUTHORITY_ID)
+    if (authority == null) authority = "microsoft.com"
```

Comment:
Should we also set it to the default of "microsoft.com" if the authority has a value but it's empty or just whitespace?